### PR TITLE
Fixed tabBarUnderlineHeight property

### PR DESCRIPTION
**Pull request by stereoki**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -209,6 +209,9 @@ const ScrollableTabView = React.createClass({
     if (this.props.tabBarUnderlineColor) {
       tabBarProps.underlineColor = this.props.tabBarUnderlineColor;
     }
+    if (this.props.tabBarUnderlineHeight) {
+      tabBarProps.underlineHeight = this.props.tabBarUnderlineHeight;
+    }
```
Comment (skv-headless) [inline index.js]: I don't want to pass property through ```index.js``` in future I will remove properties like ```tabBarUnderlineColor``` from there

**Comment by artyom-dorokhov** [comment]
ok. however component should be fully customisable.

**Comment by stereoki** [comment]
ok. however component should be fully customisable.

**Comment by skv-headless** [comment]
absolutely agree. Thats the best way I think
```<ScrollableTabView renderTabBar={() => <DefaultTabBar underlineHeight={10} />}>```

**Comment by skv-headless** [comment]
thanks for PR I've added this property in master

**Pull request closed without merging**